src/Ryujinx.HLE/HOS/Services/Fs/FileSystemProxy/ILazyFile.cs

```diff
@@ -0,0 +1,65 @@
+using LibHac;
+using LibHac.Common;
+using LibHac.Fs;
+using System;
+using System.IO;
+
+namespace Ryujinx.HLE.HOS.Services.Fs.FileSystemProxy
+{
+    public class ILazyFile : LibHac.Fs.Fsa.IFile
```

Comment:
```suggestion
    class ILazyFile : LibHac.Fs.Fsa.IFile
```
I think this should be called just `LazyFile` since it's not an interface.